Cython/StringIOTree.py

```diff
@@ -66,7 +66,13 @@ def copyto(self, target):
             child.copyto(target)
         stream_content = self.stream.getvalue()
         if stream_content:
-            target.write(stream_content)
+            try:
+                target.write(stream_content)
+            except UnicodeEncodeError as e:
```

Comment:
What is this used for?

Reply:
This was debugging code that was accidentally left in. Removed now